CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 ## Prerelease
 
+### Updates
+
+- [Updates `Template` component(s)](<(https://newyorkpubliclibrary.atlassian.net/browse/DSD-1884)>) and styles per TAD.
```

Comment:
Did you mean to have the internal `<( ` and `)>`?

Reply:
Don't think so!